README.md

```diff
@@ -1,12 +1,16 @@
 # bmap-rs
 
-The bmap-rs project aims to implement tools related to bmap. The project is written in
-rust. The inspiration for it is an existing project that is written in python called 
-[bmap-tools](https://salsa.debian.org/debian/bmap-tools). 
+## Introduction
+
+`bmap-rs` is a generic tool for copying files using the block map. The idea is that
+large files, like raw system image files, can be copied or flashed a lot faster and
+more reliably with `bmap-rs` than with traditional tools, like `dd` or `cp`. The
+project is written in rust. The inspiration for it is an existing project that is
+written in python called [bmap-tools](https://salsa.debian.org/debian/bmap-tools).
 
```

Comment:
it may be good to have a sentence to just explain why we did the rewrite; something like, we want to create smaller disk images without python depencencies